frontend/src/metabase/common/utils/columns.ts

```diff
@@ -37,5 +37,5 @@ export function getColumnIcon(column: Lib.ColumnMetadata): IconName {
     return "int";
   }
 
-  return "unknown";
+  return "list";
```

Comment:
I think we can do better, let's ask design team for help.

Reply:
That's what they suggested